src/field.js

```diff
@@ -79,7 +79,12 @@ function computeFieldObject (data, template, options) {
   if (!metadata.fields && data.statistics) return computeFields(data.statistics[0], template, options).fields
   else if (!metadata.fields) return computeAggFieldObject(data, template, options)
 
-  const fields = metadata.fields.map(field => {
+  var outFields = options.outFields.split(",")
```

Comment:
@dmfenton Are you saying if the request contains no ``` outFields ```, you want Feature Server to return the full payload of fields in the response?  I can /will check for null but want to know what fields should be returned if outFields is null ?